Kickstarter-iOS/Views/Controllers/LoginViewController.swift

```diff
@@ -33,9 +33,11 @@ internal final class LoginViewController: UIViewController {
     self.emailTextField.addTarget(self,
                                   action: #selector(emailTextFieldDoneEditing),
                                   for: .editingDidEndOnExit)
+
     self.emailTextField.addTarget(self,
                                   action: #selector(emailTextFieldChanged(_:)),
-                                  for: .editingChanged)
+                                  for: [.editingDidEndOnExit, .editingChanged])
+
```

Comment:
Curious, have you run this in tests? Wondering if anything would behave differently now that both inputs will be fired on exit.